docs/content/en/http-methods.md

```diff
@@ -5,54 +5,272 @@ position: 9
 category: API
 ---
 
-<alert type="tip">
+<alert type="info">
 
-See [here](/guide/usage.html#making-requests) for usage information for below methods.
+See [here](/usage#making-requests) for usage information for below methods.
 
 </alert>
 
-<alert type="tip">
+<alert type="info">
 
 Each http method returns a `Promise`
 
 </alert>
 
-### `delete`
-### `get`
+### `$head`
+- arguments: `(url, options?)`
+  - url: `String`
+  - options: [options](/options)
+- resolves: [JSON](https://developer.mozilla.org/en-US/docs/Web/API/Body/json)
```

Comment:
Actually, every HTTP request resolved to either text or JSON, if the text returned can be parsed as JSON, it will be converted automatically, see https://github.com/nuxt/http/blob/a5ee306a893b2e2733f413e25dbcafcb16a62740/lib/plugin.js#L125

We use [destr](https://github.com/nuxt-contrib/destr) internally to smart parse the response.

Reply:
In fact, but in case of it's not a JSON response, what would be that ? Just a simple Response so ?
I was think that it will always be JSON (in case of success) or Error in other hand

Reply:
Actually we use [destr](https://github.com/nuxt-contrib/destr) package for json! It is smart enough to parse native values or fallback to string :)

Reply:
Yeah so what we need to precise as response 🤔 I don't know at the moment 😂